parsec/backend/pki.py

```diff
@@ -31,7 +52,7 @@
     AuthenticatedClientContext,
     AnonymousClientContext,
 )
-from parsec.backend.utils import api, catch_protocol_errors
+from parsec.backend.utils import api, api_typed_msg_adapter, catch_protocol_errors, ClientType
```

Comment:
We should remove the re-export of `ClientType` from `backend.utils` to avoid cyclic import

Reply:
Can you import from `_parsec`

Reply:
Done.